en/components/chip.md

```diff
@@ -164,13 +191,71 @@ Chip rendering can be customized so its dimensions can be different using the [`
 </igx-chip>
 ```
 
-A few examples of how the chip can be even further customized:
+You can customize the `select icon`, using the [`selectIcon`]({environment:angularApiUrl}/classes/igxchipcomponent.html#selecticon) input. It accepts values of type `TemplateRef` and overrides the default icon while retaining the same functionality.
+
+![Custom Selection](../images/chip/selecting_custom.gif)
+
+```html
+<igx-chip *ngFor="let chip of chipList" [selectable]="true" [selectIcon]="mySelectIcon">
+    <igx-icon igxPrefix>{{chip.icon}}</igx-icon>
+    {{chip.text}}
+</igx-chip>
+
+<ng-template #mySelectIcon>
+    <igx-icon>check_circle</igx-icon>
+</ng-template>
+```
+
+You can customize the `remove icon`, using the [`removeIcon`]({environment:angularApiUrl}/classes/igxchipcomponent.html#removeicon) input. It takes a value of type `TemplateRef` and renders it instead of the default remove icon.
+
+![Remove icons](../images/chip/remove_icons.jpg)
+
+```html
+<igx-chip *ngFor="let chip of chipList"  [removable]="true" [removeIcon]="myRemoveIcon">
+    <igx-icon igxPrefix>{{chip.icon}}</igx-icon>
+    {{chip.text}}
+</igx-chip>
+
+<ng-template #myRemoveIcon>
+    <igx-icon>delete</igx-icon>
+</ng-template>
+```
+
+### Chip Area
+
+The [`IgxChipsAreaComponent`]({environment:angularApiUrl}/classes/igxchipsareacomponent.html) is used, when handling more complex scenarios that require interaction between chips (dragging, selection, navigation etc.).
+
+#### Reorder Chips
+
+![Dragging](../images/chip/dragging.gif)
+
+The chip can be dragged by the end-user in order to change it's position. The dragging is disabled by default, but can be enabled, using [`draggable`]({environment:angularApiUrl}/classes/igxchipcomponent.html#draggable) input. You need to handle the actual moving of the chip in the template manually. This is where the chip area comes in handy since it provides the [`onReorder`]({environment:angularApiUrl}/classes/igxchipsareacomponent.html#onreorder) event that returns the new order if a chip is dragged over another chip to take its place.
```

Comment:
```suggestion
The chip can be dragged by the end-user in order to change its position. The dragging is disabled by default but can be enabled using the [`draggable`]({environment:angularApiUrl}/classes/igxchipcomponent.html#draggable) input property. You need to handle the actual chip reordering manually. This is where the chip area comes in handy since it provides the [`onReorder`]({environment:angularApiUrl}/classes/igxchipsareacomponent.html#onreorder) event that returns the new order when a chip is dragged over another chip.
```